### PR TITLE
Removed static slidesPerView Configuration

### PR DESCRIPTION
**Pull request by dqdquint**
## Description

Tested with shop no: 31516 and used config settings deployed for Pollin: 12251

## Type of change

- [ ] Bug fix
- [x] New feature
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I updated the CHANGELOG.md

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,11 @@
 ### Fixed
 ### Removed
 
+## [3.0.1]
```
Comment (aaron-martin) [inline CHANGELOG.md:11]: There is a Unreleased section that includes a change. That change should be included in the change log entry for 3.0.1.

File: CHANGELOG.md [hunk 2]
```diff
@@ -50,6 +55,7 @@ First version of the extension.
 Initial commmit.
 
 [Unreleased]: https://github.com/shopgate/ext-upselling/compare/v2.0.0...HEAD
+[3.0.1]: https://github.com/shopgate/ext-upselling/compare/v2.0.1...v3.0.1
```
Comment (aaron-martin) [inline CHANGELOG.md]: this should be https://github.com/shopgate/ext-upselling/compare/v3.0.0...v3.0.1. There is a v3.0.0 tag

**Review by aaron-martin — COMMENTED** [review]
The slider on the product detail page shows one slide per view. The version number is not updated in the extension-config.json

**Review by aaron-martin — CHANGES_REQUESTED** [review]
request changes

File: CHANGELOG.md (hunk 1 above)
Comment (dqdquint) [reply CHANGELOG.md:11]: Updated. Thanks!

File: CHANGELOG.md (hunk 2 above)
Comment (dqdquint) [reply CHANGELOG.md]: updated!

**Review by aaron-martin — CHANGES_REQUESTED** [review]
See if you can get the tests running successfully in travis.

**Review by aaron-martin — APPROVED** [review]

**Pull request merged**